whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/FullTextSearchNoIndexTestIntegration.java

```diff
@@ -2,19 +2,19 @@
 
 import net.ripe.db.whois.api.AbstractIntegrationTest;
 import net.ripe.db.whois.api.RestTest;
-import net.ripe.db.whois.common.IntegrationTest;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
+
+import org.junit.jupiter.api.BeforeAll;
+
+import org.junit.jupiter.api.Test;
 
 import javax.ws.rs.NotFoundException;
 
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.fail;
 
```

Comment:
Can you import the ```org.junit.jupiter.api.Tag``` class, so you can annotate using ```@Tag``` instead of specifying the whole package?

Reply:
We already have a Tag class in net.ripe.db.whois.common.domain. It was causing Ambiguous  references in Quite lot of classes.

Reply:
OK! Thanks